### PR TITLE
Stub out a bunch of new chains

### PR DESCRIPTION
**Pull request by jumpsiegel**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/1357)

### CODE REVIEW

File: sdk/js/src/utils/consts.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ export const CHAINS = {
   moonbeam: 16,
   neon: 17,
   terra2: 18,
+  aptos: 20,
```
Comment (evan-gray) [inline sdk/js/src/utils/consts.ts:21]: I think 20 might be reserved for Osmosis but the PR hasn’t landed yet

**Review by evan-gray — CHANGES_REQUESTED** [review]
Might have to bump chain ids one, and don’t forget to update the changelog when updating the SDK 🙂

File: sdk/js/src/utils/consts.ts (hunk 1 above)
Comment (jumpsiegel) [reply sdk/js/src/utils/consts.ts:21]: Then I win?  :)

Comment (jumpsiegel) [reply sdk/js/src/utils/consts.ts]: Updated to 22

**Review by kcsongor — DISMISSED** [review]

**Review by kcsongor — APPROVED** [review]

**Review by SEJeff — DISMISSED** [review]
Update the README before merging please.

**Review by SEJeff — DISMISSED** [review]

**Review by evan-gray — APPROVED** [review]

**Pull request merged**